lib/workers/repository/update/branch/auto-replace.ts

```diff
@@ -179,6 +177,8 @@ export async function doAutoReplace(
     const depIndex = existingContent.indexOf(depName!);
     const valIndex = existingContent.indexOf(currentValue!);
     searchIndex = depIndex < valIndex ? depIndex : valIndex;
+    // } else if (upgrade.replacementPrefix) {
```

Comment:
?

Reply:
🙈 sorry - not sure how this got onto this branch - 🧹 removed